src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -501,6 +501,11 @@ private void HasBaseInterfaceDeclaringInterface(NamedTypeSymbol baseInterface, N
             BindingDiagnosticBag diagnostics,
             CancellationToken cancellationToken)
         {
+            if (this.BaseTypeNoUseSiteDiagnostics?.IsErrorType() == true)
```

Comment:
>if (this.BaseTypeNoUseSiteDiagnostics?.IsErrorType() == true) [](http://example.com/codeflow?start=12&length=61)

Consider adding a comment explaining the motivation to suppress diagnostics, it is safe to execute the code below under this condition.